### DISCUSSION TITLE
Allow usage of electric fields #42

### DISCUSSION THREAD

**Discussion by tobigithub** [qcxms/QCxMS#43] [General]
Hi @awvwgk ,
it would be interesting to have a short explanation of this commit #42 to "Allow usage of electric fields".
How does it work, how is it implemented? Nothing too complicated but maybe in layman's terms?

Background of course, that in mass spectrometry electric and magnetic fields are used to focus, 
accelerate and decelerate ions. So this is very interesting.

Thanks
Tobias

**Comment by JayTheDog** [comment]
For now, this is something we consider to implement, but should not directly make the program much different. In fact, there should not be a difference to the way the energy is scaled in the current code. However, I will check how this will influence the behavior of the CID mode.
